client/apollo/react/src/Icon/IconCommon.tsx

```diff
@@ -17,30 +17,35 @@ export const iconSizeVariants = {
 } as const;
 export type IconSizeVariants = keyof typeof iconSizeVariants;
 
-export type IconProps = {
+export type IconProps = SVGProps<SVGSVGElement | null> & {
   icon: string;
   variant?: IconVariants;
   size?: IconSizeVariants;
-  background?: boolean;
+  hasBackground?: boolean;
 };

```

Comment:
Ce serait mieux de mettre en type les props du composant Svg comme ça le jour où les props changent ce sera automatiquement pris en compte (+ j'ai enlevé la prop `icon` pour que ce soit le src de Svg qui soit utilisé pour que les 2 soient cohérents)

```suggestion
export type IconProps = ComponentProps<typeof Svg> & {
  variant?: IconVariants;
  size?: IconSizeVariants;
  background?: boolean;
  hasBackground?: boolean;
};
```